src/webapp/js/utils/react-table-search-params.js

```diff
@@ -4,7 +4,11 @@ export function buildSearchParams(filtered, sorted, page, pageSize) {
   const searchParams = {};
 
   _.forEach(filtered, (filter) => {
-    searchParams[filter.id] = filter.value;
+    if (filter.id === 'roles[0].name') {
```

Comment:
I think this case could be handled in users-table.js, and keep the react-table-search-params more generic